### PR TITLE
add /processor endpoints

### PR DESCRIPTION
**Pull request by evanlimanto**
- https://plaid.com/docs/link/stripe/#step-4-write-server-side-handler
- https://plaid.com/docs/link/dwolla/#step-4-write-server-side-handler

### CODE REVIEW

**Comment by evanlimanto** [comment]
@ntindall can you review this? i'm not in the org yet, so can't assign you.

File: plaid/processor_test.go [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+	accountID := accountsResp.Accounts[0].AccountID
+
+	apexTokenResp, err := testClient.CreateApexToken(tokenResp.AccessToken, accountID)
+	assert.Nil(t, err)
+	assert.True(t, strings.HasPrefix(apexTokenResp.ProcessorToken, "processor-sandbox-"))
+
+}
+
+func TestCreateDwollaToken(t *testing.T) {
+	sandboxResp, _ := testClient.CreateSandboxPublicToken(sandboxInstitution, testProducts)
```
Comment (ntindall) [inline plaid/processor_test.go]: shall we check the error ?

File: plaid/processor_test.go [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+	accountsResp, err := testClient.GetAccountsWithOptions(tokenResp.AccessToken, options)
+	accountID := accountsResp.Accounts[0].AccountID
+
+	dwollaTokenResp, err := testClient.CreateDwollaToken(tokenResp.AccessToken, accountID)
+	assert.Nil(t, err)
+	assert.True(t, strings.HasPrefix(dwollaTokenResp.ProcessorToken, "processor-sandbox-"))
+}
+
+func TestCreateStripeToken(t *testing.T) {
+	sandboxResp, _ := testClient.CreateSandboxPublicToken(sandboxInstitution, testProducts)
```
Comment (ntindall) [inline plaid/processor_test.go]: lets check the error

File: plaid/processor.go [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+}
+
+type createStripeTokenRequest struct {
+	ClientID    string `json:"client_id"`
+	Secret      string `json:"secret"`
+	AccessToken string `json:"access_token"`
+	AccountID   string `json:"account_id"`
+}
+
+type createStripeTokenResponse struct {
```
Comment (ntindall) [inline plaid/processor.go]: we need to export this.

File: plaid/processor.go [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+}
+
+type createDwollaTokenRequest struct {
+	ClientID    string `json:"client_id"`
+	Secret      string `json:"secret"`
+	AccessToken string `json:"access_token"`
+	AccountID   string `json:"account_id"`
+}
+
+type createDwollaTokenResponse struct {
```
Comment (ntindall) [inline plaid/processor.go]: we need to export this struct.

File: plaid/processor_test.go [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+import (
+	"strings"
+	"testing"
+
+	assert "github.com/stretchr/testify/require"
+)
+
+func TestCreateApexToken(t *testing.T) {
+	sandboxResp, _ := testClient.CreateSandboxPublicToken(sandboxInstitution, testProducts)
+	tokenResp, _ := testClient.ExchangePublicToken(sandboxResp.PublicToken)
```
Comment (ntindall) [inline plaid/processor_test.go]: lets check these errors

File: plaid/processor_test.go (hunk 1 above)
Comment (evanlimanto) [reply plaid/processor_test.go]: we're already checking in plaid/sandbox_test.go — do we still need to check here?

File: plaid/processor.go (hunk 3 above)
Comment (evanlimanto) [reply plaid/processor.go]: ⚡️

File: plaid/processor.go (hunk 4 above)
Comment (evanlimanto) [reply plaid/processor.go]: ⚡️

File: plaid/processor_test.go (hunk 2 above)
Comment (evanlimanto) [reply plaid/processor_test.go]: same

File: plaid/processor_test.go (hunk 5 above)
Comment (evanlimanto) [reply plaid/processor_test.go]: same

File: plaid/processor_test.go (hunk 1 above)
Comment (ntindall) [reply plaid/processor_test.go]: I think I meant something along the lines of:

"if the create sandbox public token request fails, then we are a going to get a null pointer error later in the test unless we check the error here"

does that make sense?

Comment (evanlimanto) [reply plaid/processor_test.go]: makes sense ⚡️

File: plaid/processor_test.go (hunk 2 above)
Comment (evanlimanto) [reply plaid/processor_test.go]: ⚡️

File: plaid/processor_test.go (hunk 5 above)
Comment (evanlimanto) [reply plaid/processor_test.go]: ⚡️

**Pull request merged**